testing/bobcat/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Contributor: Shiz <hi@shiz.me>
+# Maintainer: Shiz <hi@shiz.me>
+pkgname=bobcat
+pkgver=4.07.00
+pkgrel=0
+pkgdesc="library of C++ classes and templates handling child processes, streams/sockets, shared memory, config files, etc."
+url="https://fbb-git.github.io/bobcat/"
+arch="all"
+license="GPL2+"
+makedepends="icmake bash yodl"
+subpackages="$pkgname-doc $pkgname-dev"
+options="!check"
```

Comment:
When disabling `check`, please add a short comment why, e.g. `options="!check"  # upstream does not provide tests`.